src/System.Private.DataContractSerialization/src/System/Xml/ValueHandle.cs

```diff
@@ -475,11 +475,7 @@ public byte[] ToByteArray()
 
         public string GetString()
         {
-            ValueHandleType type = _type;
-            if (type == ValueHandleType.UTF8)
```

Comment:
This code might be written for perf optimization. We'd better not remove this without perf measurement.

/cc: @mconnew